espflash/src/command.rs

```diff
@@ -133,38 +151,45 @@ pub enum Command<'a> {
     SpiAttachStub {
         spi_params: SpiAttachParams,
     },
-    ChangeBaud {
+    ChangeBaudrate {
         /// New baud rate
         new_baud: u32,
         /// Prior baud rate ('0' for ROM flasher)
         prior_baud: u32,
     },
-    FlashDeflateBegin {
+    FlashDeflBegin {
         size: u32,
         blocks: u32,
         block_size: u32,
         offset: u32,
         supports_encryption: bool,
     },
-    FlashDeflateData {
+    FlashDeflData {
```

Comment:
Any particular reason to rename these?

Reply:
As you mentioned, I updated them to align them with `esptool` and its [documentation](https://docs.espressif.com/projects/esptool/en/latest/esp32/advanced-topics/serial-protocol.html#commands)